protocols/mdns/Cargo.toml

```diff
@@ -25,8 +24,26 @@ smallvec = "1.6.1"
 socket2 = { version = "0.4.0", features = ["all"] }
 void = "1.0.2"
 
+async-io-crate  = { package = "async-io", version = "1.3.1", optional = true }
+tokio-crate = { package = "tokio", version = "1.19", default-features = false, features = ["net", "time"], optional = true}
+
+[features]
+default = ["async-io"]
+tokio = ["tokio-crate"]
+async-io = ["async-io-crate"]
```

Comment:
I think it would be good to use the `dep:` syntax here to hide the other two features.

Reply:
@thomaseizinger  Do you mean use `tokio = ["dep:tokio"]` instead of `tokio = ["tokio-crate"]`?